plugin/pkg/scheduler/scheduler.go

```diff
@@ -50,12 +52,14 @@ type Scheduler struct {
 	config *Config
 }
 
+// StopEverything when called, closes the StopEverything channel to shut down
+// the Scheduler.
 func (sched *Scheduler) StopEverything() {
 	close(sched.config.StopEverything)
 }
 
-// These are the functions which need to be provided in order to build a Scheduler configuration.
-// An implementation of this can be seen in factory.go.
+// Configurator is comprised of the functions needed in order to build a
```

Comment:
nit: `Configurator defines I/O, caching, and other functionality needed to construct a new scheduler`  okay to keep as-is also, just a suggestion.